bin/pycbc_make_sky_grid

```diff
@@ -0,0 +1,103 @@
+    spher[:,1] = np.arcsin(sky_points[:,2])
+    return spher
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument('--ra', type=float, help="Right ascension (in rad) of the center of the GRB error box")
+parser.add_argument('--dec', type=float, help="Declination (in rad) of the center of the GRB error box")
+parser.add_argument('--instruments', nargs="+", type=str, required=True, help="List of instruments to analyze.")
+parser.add_argument('--sky-error', type=float, required=True, help="3-sigma confidence radius (in rad) of the external trigger error box")
+parser.add_argument('--trigger-time', type=int, required=True, help="Time (in s) of the external trigger")
+parser.add_argument('--timing-uncertainty', type=float, default=0.0001, help="Timing uncertainty (in s) we are willing to accept")
+parser.add_argument('--output', type=str, required=True, help="Name of the sky grid")
+
+args = parser.parse_args()
+
+if len(args.instruments) == 1:
+    parser.error('Can not make a sky grid for only one detector.')
+
+args.instruments.sort() # Put the ifos in alphabetical order
+detectors = args.instruments
+detectors = [Detector(d) for d in detectors]
+detector_pairs = list(itertools.combinations(detectors, 2))
+
+# Select the detector pair for which the target location has smallest relative arrival time difference
+tds = [detector_pairs[i][0].time_delay_from_detector(detector_pairs[i][1], args.ra, args.dec, args.trigger_time) for i in range(len(detector_pairs))]
+smallest_td = np.argmin(tds)
+
+# Calculate the light travel time between the two detectors
+t = detector_pairs[smallest_td][0].light_travel_time_to_detector(detector_pairs[smallest_td][1])
+
+# Calculate the required angular spacing of the sky points
+angular_spacing = (2*args.timing_uncertainty) / np.sqrt(t**2 - tds[smallest_td]**2)
+
+sky_points = np.zeros((1, 2))
+
+number_of_rings = int(args.sky_error / angular_spacing)
+
+# Generate the sky grid centered at the North pole
+for i in range(number_of_rings+1):
+    if i == 0:
+        sky_points[0][0] = 0
+        sky_points[0][1] = np.pi/2
+    else:
+        number_of_points = int(2*np.pi*i)
+        for j in range(number_of_points):
+            sky_points = np.row_stack((sky_points, np.array([j/i, np.pi/2 - i*angular_spacing])))
+
+# Convert spherical coordinates to cartesian coordinates
+cart = spher_to_cart(sky_points)
+
+grb = np.zeros((1, 2))
+grb[0] = args.ra, args.dec
+grb_cart = spher_to_cart(grb)
+
+north_pole = [0, 0, 1]
+
+ort = np.cross(grb_cart, north_pole)
+norm = np.sqrt(ort[0][0]**2 + ort[0][1]**2 + ort[0][2]**2)
+ort /= norm
+n = -np.arccos(np.dot(grb_cart, north_pole))
+u = ort*n
+
+# Rotate the sky grid to the center of the external trigger error box
+r = R.from_rotvec(u)
+rota = r.apply(cart)
+
+# Convert cartesian coordinates back to spherical coordinates
+spher = cart_to_spher(rota)
+
+with h5py.File(args.output + '.hdf', 'w') as hf:
```

Comment:
Let the `.hdf` extension be given from the command line, do not add it explicitly. Sometimes people use `.h5` instead.